src/curator/pages/home/views.cljs

```diff
@@ -24,48 +27,114 @@
           [:div.dropdown-menu {:id "dropdown-menu" :role "menu"}
            [:div.dropdown-content.has-text-left
             [:a.dropdown-item {:href "#"
-                               :on-click #(dispatch [:home/set-search-topic :gene])} 
+                               :on-click #(dispatch [:home/set-search-topic :gene])}
              "gene"]
             [:a.dropdown-item {:href "#"
-                               :on-click #(dispatch [:home/set-search-topic :disease])} 
+                               :on-click #(dispatch [:home/set-search-topic :disease])}
              "disease"]
             [:a.dropdown-item {:href "#"
                                :on-click #(dispatch [:home/set-search-topic :affiliation])}
-             "affiliation"]]]]]
+             "affiliation"]
+            [:a.dropdown-item {:href "#"
+                               :on-click #(dispatch [:home/set-search-topic :variation])}
+             "variation"]
+            ]]]]
+        ; two fifths for search bar
         [:div.column.is-two-fifths
          [:div.field
           [:div.control
            [:input.input {:id "search-box"
                           :type "text"
                           :placeholder topic
                           :on-change #(dispatch [:home/request-suggestions
-                                                 (-> % .-target .-value)])}]]]
-         (when (seq suggested-genes)
-           [:div.box
-            (for [gene suggested-genes]
-              [:div.block
-               [:a {:href (rfe/href :genes gene)}
-                (:text gene)]])])]]]]]))
+                                                 ;(fn [event] (println event) (-> event .-target .-value))
```

Comment:
I think we'll probably want to split the variation display onto a separate page soon--but this is a starting point.